WolvenKit.App/Services/PluginService.cs

```diff
@@ -103,6 +105,34 @@ public void Init()
                         Status = EPluginStatus.NotInstalled
                     });
                 }
+
+
+                // Redmod
+                if (id == EPlugin.redmod)
+                {
+                    var plugin = Plugins.FirstOrDefault(x => x.Id == id);
+                    var redModLocalversion = "";
```

Comment:
Define the variables where you use them, for example this variable is not used in the `if (id == EPlugin.redmod)`, but it is used in the `if (File.Exists(redModManifest))` scope.

I think it is more nicer to have them defined where they are actually used, unlike C 🏃‍♀️